app/models/miq_request_workflow.rb

```diff
@@ -200,6 +200,10 @@ def validate(values)
           # TODO: do we need validation for boolean
         when :button
           # Ignore
+        when :array_integer
+          unless is_array?(value)
```

Comment:
I realize you are following a pattern here, but rubocop (🤖 👮 ) is not happy.  Should this just be `value.kind_of?(Array)` instead.